src/intent.html

```diff
@@ -221,20 +221,14 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
    but as with any properties, is by default ignored but may have a
    system-specific effect.</p>
 
-    <p>As with [=Concepts=], The Working group maintains two lists
-      of [=property=] values.</p>
+    <p>Similar to the lists of [=Concepts=], The Working Group maintains a list of [=property=] values that should be implemented.</p>
     <ul>
-      <li><dfn id="intent-core-properties">Core properties</dfn>: This is a
-      <a href="https://w3c.github.io/mathml-docs/intent-core-properties">list core of properties</a>
+      <li><dfn id="intent-core-properties">Properties list</dfn>: This is the full
```

Comment:
The id could be updated (while this would break permanent links)

Reply:
yes I left this, the correspondence with the core concept list is till there even though we decided (for whatever reason) to drop the open list for properties.

Reply:
> we decided (for whatever reason) 

The reason was removing process friction in deciding whether new additions belong in Core or Open, as it was hard to have clear consensus on the bar for entering either list.

Clarifying, since it appeared necessary.

Reply:
certainly we are doing it to remove process friction, I don't think anyone understood why you think it does, but as I say, if it does and we can move on, then that's all good. Since the open list (should have been) empty anyway it's existence shouldn't have been contentious, but conversely its loss isn't any great loss.

Reply:
> I don't think anyone understood why you think it does

That's rather uncharitable and unneeded, I take your point.

Reply:
@dginev I can delete the comment if you wish, but  I'm not sure why you say it is uncharitable, I'd say it was  fairly accurate sentiment of the meeting. the open properies list is being removed not on any technical grounds but simply because you said removing it would remove the process block that meant we could not progress on the spec without spending weeks discussing each property and revisiting 2 year old decisions. As has been discussed, the list should not have contained any property that was being used in public documents so removing it is not really any loss, so definitely a price worth paying.